### PR TITLE
Javascript javascript3 week3

### PR DESCRIPTION
**Pull request by dan-nil**
Homework Javascript 3 - Week 3

### CODE REVIEW

File: javascript/javascript3/week3/website-screenshot-app/script.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      displayContainer.innerHTML = '<h1>No data to display</h1>';
+    }
+  } catch (err) {
+    console.log(err.statusText);
+  }
+}
+
+btnSubmit.addEventListener('click', async (e) => {
+  e.preventDefault();
+  const website = websiteInput.value;
```
Comment (shpomp) [inline javascript/javascript3/week3/website-screenshot-app/script.js]: This could also be up top with other const 💡

**Review by shpomp — APPROVED** [review]
Looks great! 🌟 
I did not find much to cling to. 
Sorry it took so long to get it reviewed, it slipped through somehow 🙈 🥺

File: javascript/javascript3/week3/website-screenshot-app/script.js (hunk 1 above)
Comment (dan-nil) [reply javascript/javascript3/week3/website-screenshot-app/script.js]: It was fixed.

Comment (dan-nil) [reply javascript/javascript3/week3/website-screenshot-app/script.js]: Thank you for the review!

**Pull request merged**